### DISCUSSION TITLE
Hyprland Hall of Fame

### DISCUSSION THREAD

**Discussion by T-Crypt** [prasanthrangan/hyprdots#204] [Show and tell]
its time to submit these dotfiles to the Hall of fame. You are the top dotfiles repo in Hyprland Topics for github!! -- Also need to get featured on Hyprlands repo 

https://github.com/topics/hyprland 

Need to submit via Discord -- Should be put up for December of 2023 
https://hyprland.org/rices

**Comment by prasanthrangan** [comment]
> its time to submit these dotfiles to the Hall of fame. You are the top dotfiles repo in Hyprland Topics for github!! -- Also need to get featured on Hyprlands repo
> 
> https://github.com/topics/hyprland

Hhahaa thank you so much, i didn't expect this to happen tbh!

> Need to submit via Discord -- Should be put up for December of 2023
> https://hyprland.org/rices

Oh I didn't know this was a thing, and I'm not active on discord either...
that looks cool, but i don't care much about "hall of fame" anyways...
